src/http_server.cc

```diff
@@ -2737,7 +2745,8 @@ HTTPAPIServer::HandleInfer(
   }
 
   if (err != nullptr) {
-    LOG_VERBOSE(1) << "Infer failed: " << TRITONSERVER_ErrorMessage(err);
+    LOG_VERBOSE(1) << "[request id: " << request_id << "]"
```

Comment:
Does it make more sense to use a common `GetRequestId()` for both servers in this PR too? See this comment: https://github.com/triton-inference-server/backend/pull/60/files#r912262759

Reply:
Same as other discussion, please add spaces after the "]" for consistency

Reply:
@dyastremsky

Reply:
Sure, thanks. Done.

Reply:
Sure, thanks. Done.